content/docs/iac/packages-and-automation/continuous-delivery/bitbucket.md

```diff
@@ -0,0 +1,108 @@
+---
+title_tag: "Using Bitbucket Pipelines | CI/CD"
+meta_desc: This page details how to use Bitbucket Pipelines to manage deploying staging and production stacks based on commits to specific Git branches.
+title: Bitbucket Pipelines
+h1: Pulumi CI/CD & Bitbucket
```

Comment:
Consider "Pulumi CI/CD with Bitbucket Pipelines".    Since we specify "Github Actions" instead of just "Github" in that guide.